content/_landingpage/_features/arm.md

```diff
@@ -4,4 +4,4 @@ icon: fa-terminal
 
 ### ARM
```

Comment:
Formerly used to display the parts of the homepage. I am likely to remove them altogether in the future.